AffirmSDK/AffirmClient.m

```diff
@@ -61,11 +61,13 @@ + (nullable AffirmResponse *)parseError:(NSData *)data
     if (responseObject == nil) {
         return nil;
     }
-    NSString *message = [responseObject valueForKey:@"message"];
     NSString *code = [responseObject valueForKey:@"code"];
-    NSString *type = [responseObject valueForKey:@"type"];
+    NSString *field = [responseObject valueForKey:@"field"];
+    NSString *message = [responseObject valueForKey:@"message"];
     NSNumber *statusCode = [responseObject valueForKey:@"status_code"];
-    return [[AffirmErrorResponse alloc] initWithMessage:message code:code type:type statusCode:statusCode];
+    NSString *type = [responseObject valueForKey:@"type"];
+
+    return [[AffirmErrorResponse alloc] initWithMessage:message code:code field:field type:type statusCode:statusCode];
```

Comment:
@hoyelo just had a quick question. For existing merchants who have integrated with the SDK prior to these changes - will this `AffirmErrorResponse` be backwards compatible when accessed via the existing callback (`didFailWithError:(NSError *)error`?

Reply:
Yes it is, because it's always converted to `NSError`, not `AffirmErrorResponse `.